primitives/evm/src/lib.rs

```diff
@@ -0,0 +1,39 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
do we need to fork this?

if we do, I don't think this needs to be an standalone crate, can just be part of primitives create

Reply:
@xlc can move it into `module-evm`